js/views/PageNav.js

```diff
@@ -59,6 +61,10 @@ export default class PageNav extends View {
     }
   }
 
+  navListBtnClick() {
+    this.$('.js-navList').toggleClass('open');
```

Comment:
Is this just a dummy list that will be removed at some point?

If not, then I think we should cache the selector. The page nav will live for the life of the app and it's silly to keep going to the DOM for the same element.